.travis.yml

```diff
@@ -4,6 +4,10 @@ rvm:
   - 2.2
   - 2.3
   - 2.4
+  - 2.5
+
+before_install:
+  - gem update --system # Need for Ruby 2.5.0. https://github.com/travis-ci/travis-ci/issues/8978
```

Comment:
Apparently fixed by https://github.com/travis-ci/travis-ci/issues/8969 ?

Reply:
Thanks @jeremy , you are right!
However another issue has came and I updated before install script.
Ref: https://github.com/travis-ci/travis-ci/issues/9333